cloud/aws/templates/aws_oidc/filestorage.tf

```diff
@@ -0,0 +1,52 @@
+resource "aws_s3_bucket" "civiform_files_s3" {
+  bucket = "civiform-files-s3"
```

Comment:
question (blocking): should this be parameterized so that the same AWS account can run multiple deployments?